Tests/iaas/flavor-naming/flavor-name-check.py

```diff
@@ -217,9 +251,10 @@ def out(self):
                     i += 3
                     lst.append(att)
                 par += 1
-            elif self.outstr[i+1] == ":":
+            elif self.outstr[i+1] == ":":   # change?
                 if att:
-                    ostr += ":%"
+                    # ostr += ":%"
```

Comment:
imho remove, if left in commented out, imho leave comment why it is left there, so it clear why it was left there.

Reply:
Fair comment, this has changed from v1 to v2. But git history is good enough if we ever want to do archeology.